app/graphql/mutations/term/create_term.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class Mutations::Term::CreateTerm < Mutations::Term::BaseMutation
+  argument :vocabulary_string_key, ID, required: true
+  argument :pref_label, String, required: true
+  argument :alt_labels, [String], required: false
+  argument :authority, String, required: false
+  argument :uri, String, required: false
+  argument :term_type, Types::TermCategory, required: false # enum local, temporary, external
+  argument :custom_fields, [Types::CustomFieldAttributes], required: false
+
+  field :term, Types::TermType, null: true
```

Comment:
When would this field return null?

Reply:
When there is an error with the request or if the object isn't found.

Reply:
I just tested things out in a mutation in my local copy to confirm, and it looks like any error raised will short circuit the rest of the resolve logic and you won't actually end up returning that field at all (null or otherwise).

Here's an example with the updateUser mutation, where I'm sending a user id that isn't valid:

```
mutation {
  updateUser(input: {id:"does-not-exist"}){
    user {
      id
    }
  }
}
```

The response I get is:

```
{
  "data": {
    "updateUser": null
  },
  "errors": [
    {
      "message": "Couldn't find User",
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ],
      "path": [
        "updateUser"
      ]
    }
  ]
}
```

I can do `field :user, Types::UserType, null: true` or `null: false` in that mutation and it doesn't make a difference, so that's why I thought that setting `null: true` for a field was a way of saying "there are cases where I *won't* raise an error that graphql will catch for the entire request (and put in the top level errors array), but I still want this particular field to return null".  Do you see what I mean?  Maybe there's something here that I'm not getting, but I wanted to make sure we're on the same page.  I didn't think you'd ever see `{ createTerm: { user: null } }` when an error is raised.  I thought you'd get `{createTerm: null}` instead.

Reply:
I see what you're saying. I based my implementation on the mutation examples on the graphql-ruby site and they handle errors differently then we do for mutations. So you're right, the null parameter should be removed it. I almost think this should be a ticket because this isn't the only place where I did that.

Reply:
I am 👍 to making the null param removals another ticket.

Reply:
Ticket sounds good!  We can definitely do post-merge.

Reply:
HYACINTH-457